### DISCUSSION TITLE
How to allow users to request /users/me when they're not confirmed yet

### DISCUSSION THREAD

**Discussion by gfpacheco** [strapi/strapi#5888] [Legacy Topics]
Currently the user gets:

```
{"statusCode":401,"error":"Unauthorized","message":"Your account email is not confirmed."}
```

I want to allow them to read their profile when they didn't confirm the email yet.

I think this behavior implementation is deep in the permissions: https://github.com/strapi/strapi/blob/master/packages/strapi-plugin-users-permissions/config/policies/permissions.js#L58.

Is there any way other than copying this file and editing in my `extensions/users-permissions/config/policies/permissions.js` file?

**Comment by lauriejim** [comment]
Hello @gfpacheco !
It's possible to override this file by using the [customization concept](https://strapi.io/documentation/3.0.0-beta.x/concepts/customization.html)
In your case, you will have to add an exception on `ctx.request.url` (or something like this)

**Reply by gfpacheco** [reply]
> Is there any way other than copying this file and editing in my extensions/users-permissions/config/policies/permissions.js file?
